### PR TITLE
Fix wrong postgres type for SqlInt64

### LINKED ISSUE #75: Latest persistent fails tests
Tests are failing with latest Persistent, example is here
https://github.com/coingaming/persistent-migration
Any ideas how to fix it?
```shell
persistent-migration-integration
  postgresql
    goldens
      Migrate from empty:              NOTICE:  CREATE TABLE will create implicit sequence "persistent_migration_id_seq" for serial column "persistent_migration.id"
NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "persistent_migration_pkey" for table "persistent_migration"
NOTICE:  relation "persistent_migration" already exists, skipping
NOTICE:  CREATE TABLE will create implicit sequence "city_id_seq" for serial column "city.id"
NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "city_pkey" for table "city"
NOTICE:  CREATE TABLE / UNIQUE will create implicit index "unique_city" for table "city"
NOTICE:  CREATE TABLE will create implicit sequence "person_id_seq" for serial column "person.id"
NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "person_pkey" for table "person"
NOTICE:  CREATE TABLE will create implicit sequence "persistent_migration_id_seq" for serial column "persistent_migration.id"
FAIL (0.07s)
        user error (More migrations detected:
         * ALTER TABLE "person" ALTER COLUMN "id" TYPE INT8;
         * ALTER TABLE "city" ALTER COLUMN "id" TYPE INT8;
        )
```

### PR DESCRIPTION
**Pull request by tfc**
This is an attempt to fix https://github.com/brandonchinn178/persistent-migration/issues/75

I am not sure if the unit test fixes are correct, as i don't see how to run them locally without failing, as they even do fail on master. I tried best effort, welcoming comments on that.

cc @kaldonir @brandonchinn178

### CODE REVIEW

**Comment by brandonchinn178** [comment]
Sorry about that! The last PR seemed to break CI and I didn't notice. Can you rebase against master?

**Comment by tfc** [comment]
@brandonchinn178 rebased! :+1:

**Comment by tfc** [comment]
@brandonchinn178 great, this worked and everything is green.

After you merged, what do you think when will the next release hit hackage?

File: test/integration/Migration.hs [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DerivingStrategies #-}
 {-# LANGUAGE ExistentialQuantification #-}
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
```
Comment (brandonchinn178) [inline test/integration/Migration.hs:7]: are these still needed?

Comment (tfc) [reply test/integration/Migration.hs:7]: just checked. Yes, they are still needed.

Comment (brandonchinn178) [reply test/integration/Migration.hs:7]: oh i see. i'm not running CI on the latest GHC version

**Comment by brandonchinn178** [comment]
actually, never mind about the CHANGELOG. I'll do it

**Comment by brandonchinn178** [comment]
https://hackage.haskell.org/package/persistent-migration-0.3.0

**Pull request merged**